examples/configObject.mjs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as dotenv from "dotenv";
+dotenv.config()
+
+/**
+ * This example demostrates how to construct a configuration object from settings loaded from Azure App Configuration.
+ * If you are using configuration object instead of Map-styled settings, it would minimize the code changes required to use Azure App Configuration in your application.
+ *
+ * When you import configuration into Azure App Configuration from a local .json file, the keys are automatically flattened with specified separator (default is ".").

```

Comment:
>  (default is ".")

We dont have any default separator for import

Reply:
fixed in dc4578d